Userland/Libraries/LibGUI/Toolbar.cpp

```diff
@@ -27,6 +27,7 @@ Toolbar::Toolbar(Orientation orientation, int button_size)
     , m_button_size(button_size)
 {
     REGISTER_BOOL_PROPERTY("collapsible", is_collapsible, set_collapsible);
+    REGISTER_BOOL_PROPERTY("groupled", is_grouped, set_grouped);
```

Comment:
Typo: groupled

I'm a bit confused how this worked when the property name didn't match.

Reply:
fixed